index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="./css/general.css">
+  <title>Rohit's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div class="title">
+      <h5>Rohit Bhatt</h5>
```

Comment:
Good job 👌but it's highly recommended to use the `a` tag for your logo, so please make sure to put your logo inside the `a` tag.